tests/automation-framework/config/config.go

```diff
@@ -1,18 +1,5 @@
 package config
 
-import (
-	"encoding/json"
-	"io/ioutil"
-	"os"
-
-	"github.com/creasty/defaults"
-)
-
-var (
-	instance   *configuration = nil
-	configPath                = os.Getenv("CONFIG_PATH")
-)
-
 type RancherServerConfig struct {
```

Comment:
packages should group resources based on their usage, This struct is only used by the rancher client.  I would move it to the rancherclient pacakge and rename it to just `Config`

Reply:
OK for sure I can do that.